### PR TITLE
UXG-1269: adds omnibar search guided setup view

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UXG-1269

### CODE REVIEW

File: src/vue/backend/src/store/modules/general/actions.js [hunk 1]
```diff
@@ -13,7 +13,63 @@ const verifyAPIkey = () => {
     return new Promise(resolve => setTimeout(resolve, 5000))
 }
 
+const saveOmnibarSettings = () => {
+    // To-Do: add actual endpoints
+    // GET /idxbroker/v1/admin/settings/omnibar
```
Comment (sgreenl5) [inline src/vue/backend/src/store/modules/general/actions.js:18]: You don't need to add the api skeleton in here, just the to-do is fine. This information will not be used explicitly.

File: src/vue/backend/src/styles/formContentStyles.scss [hunk 2]
```diff
@@ -1,9 +1,19 @@
 @import '~bootstrap/scss/forms';
 .form-content {
-    display: flex;
```
Comment (sgreenl5) [inline src/vue/backend/src/styles/formContentStyles.scss:3]: In an upcoming PR, there are major updates to this file to make it correspond with the design. Be aware if this is merged in first that there will be merge conflicts.

File: src/vue/backend/src/views/guided-setup/Omnibar.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<template>
+    <GuidedSetupContentCard
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/Omnibar.vue:2]: This should be a fixed height with a scroll

File: src/vue/backend/src/templates/omnibarForm.vue [hunk 4]
```diff
@@ -1,158 +1,143 @@
 <template>
```
Comment (sgreenl5) [inline src/vue/backend/src/templates/omnibarForm.vue:1]: Page is not matching designs in terms of lack of bold titles and toggle text styles. Also the border line is too dark, should be #F2F3F4

**Review by sgreenl5 — CHANGES_REQUESTED** [review]

File: src/vue/backend/src/templates/omnibarForm.vue (hunk 4 above)
Comment (nicmarson) [reply src/vue/backend/src/templates/omnibarForm.vue:1]: I tried font-weight: 600 and it looked too bold, so I dialed it back to 500 which seemed closer to the design. I might need a ruling from the designer on which one to choose.

File: src/vue/backend/src/router/index.js [hunk 5]
```diff
@@ -105,8 +105,13 @@ const routes = [
             {
```
Comment (sgreenl5) [inline src/vue/backend/src/router/index.js:105]: This causes the compile to fail

**Review by sgreenl5 — CHANGES_REQUESTED** [review]
Compile fails due to  `guidedSetup` not being used.

**Review by sgreenl5 — APPROVED** [review]

**Review by ralls — DISMISSED** [review]
👍  Looks great

**Review by sgreenl5 — DISMISSED** [review]

**Review by sgreenl5 — DISMISSED** [review]

**Review by sgreenl5 — APPROVED** [review]

**Review by ralls — APPROVED** [review]
👍

**Pull request merged**